### PR TITLE
neat-picks on README

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

N/A

# Overview

- Remove coverall badge until we restore it
- Add a section 'how to install'
- Point 'how to build' to the wiki
- Rename 'compatibility matrix' to 'latest releases' and nest it
  under 'how to install'

# Comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +39,21 @@ The Byron version of Cardano Wallet is in the
 [cardano-sl](https://github.com/input-output-hk/cardano-sl)
 repository.
 
-## How to build
+## How to install
+
+See **Installation Instructions** for each available [releases](https://github.com/input-output-hk/cardano-wallet/releases).
```
Comment (Anviking) [inline README.md]:
```suggestion
See **Installation Instructions** for each available [release](https://github.com/input-output-hk/cardano-wallet/releases).
```

File: README.md [hunk 2]
```diff
@@ -37,13 +39,21 @@ The Byron version of Cardano Wallet is in the
...
+
+> ### Latest releases
+> 
+> | cardano-wallet                                                                            | Jörmungandr                                                                          |
```
Comment (Anviking) [inline README.md:48]: What if the table header `Jörmungandr` was `supported Jörmungandr` to highlight the relationship between the two?

Comment (Anviking) [reply README.md:48]: Additionally it _might_ be worth a sentence in "Overview" explaining _what_ Jörmungandr is.

(Imagining people landing here randomly might be unclear about what jormungandr is, and maybe people coming here for the testnet could be confused by the current generic overview which doesn't mention the testnet or jormungandr)

**Review by Anviking — APPROVED** [review]
💯

File: README.md (hunk 2 above)
Comment (KtorZ) [reply README.md:48]: You have a point! I am only seeing your comment now though :(

**Pull request merged**